### PR TITLE
Add article about Parachutes

### PR DESCRIPTION
**Pull request by jtbermido04**
Please consider adding this article to the Billboard

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -32,8 +32,8 @@
     <div class="column">
     </div>
     <div class="column">
+      <div id="example"><script>include("articles/example.md", "example")</script></div>
```
Comment (claudiodsf) [inline index.html]: Please rename everything to the actual article name

File: index.html [hunk 2]
```diff
@@ -32,8 +32,8 @@
...
     </div>
-  </div>
```
Comment (claudiodsf) [inline index.html]: Why you are removing that??

File: index.html (hunk 1 above)
Comment (jtbermido04) [reply index.html]: Renamed to parachutes

**Comment by claudiodsf** [comment]
I squashed all the commits into one and merged 😉

**Pull request merged**